main.go

```diff
@@ -14,7 +14,7 @@ import (
 )
 
 // Version is initialized at build time through -ldflags "-X main.Version=<version number>"
-var version = "1.21.0"
+var version = "1.20.0"
```

Comment:
Forgot to put it back to 1.21 (actually 1.22)

Reply:
Should I put it to 1.22.0 ?

Reply:
Yes